spec/rspec/matchers/built_in/include_spec.rb

```diff
@@ -51,6 +51,14 @@
           expect("abc\ndef").to include("g", "h")
         }.to fail_matching("expected \"abc\\ndef\" to include \"g\" and \"h\"\nDiff")
       end
+
+      it "does not diff when lines match but are not an exact match" do
+        expect {
+          expect(" foo\nbar\nbazz").to include("foo", "bar", "gaz")
+        }.to raise_error(RSpec::Expectations::ExpectationNotMetError) { |error|
+          expect(error.message).not_to match(/Diff/)
+        }
+      end
```

Comment:
This expectation can be simplified to:

``` ruby
expect {
  expect(" foo\nbar\nbazz").to include("foo", "bar", "gaz")
}.to fail_with(a_string_excluding("Diff"))
```

* `fail_with` is defined in [fail_matchers](https://github.com/rspec/rspec-expectations/blob/f0b79d1e58bcc209191bfc99a04211dd38718955/lib/rspec/matchers/fail_matchers.rb#L21-L28)
* `a_string_excluding` is defined as the negation of `a_string_including`  (which is itself an alias of `include`) -- see [here](https://github.com/rspec/rspec-expectations/blob/f0b79d1e58bcc209191bfc99a04211dd38718955/spec/support/matchers.rb#L42)